app/components/find/courses/search_result_component.rb

```diff
@@ -0,0 +1,72 @@
+module Find
```

Comment:
This is a duplicate to app/components/find_interface/results/search_result_component.rb, plus some other files...

Reply:
Which one do we keep?